_includes/sign_in.md

```diff
@@ -0,0 +1,27 @@
+For an application developer, the application flow is similar to the typical client-server flow used by centralized sign in services (e.g., OAuth). However, with Blockstack, the authentication flow happens entirely client-side.
+
+A decentralized application (DApp) and the Blockstack Browser communicate during the authentication flow by passing back and forth two tokens. The requesting application sends the Blockstack Browser an `authRequest` token. Once a user approves a sign-in, the Blockstack Browser responds to the application with an `authResponse` token. These tokens are <a href="https://jwt.io/" target="\_blank">JSON Web Tokens</a>, and they are passed via URL query strings.
+
+![](/storage/images/app-sign-in.png)
+
+When a user chooses to **Sign in with Blockstack** on a DApp, it calls the `redirectToSignIn()` method which sends an  `authRequest` to the Blockstack Browser. Blockstack passes the token in via a URL query string in the `authRequest` parameter:
+
+`https://browser.blockstack.org/auth?authRequest=j902120cn829n1jnvoa...`
+
+When the Blockstack Browser receives the request, it generates an (`authResponse`) token to the application. This token includes three key pairs:
+
+* an _ephemeral transit_ key 
```

Comment:
the ephemeral transit key and the identity-address private key aren't included in the authResponse token, however, they are used to generate data in the token.

The ephemeral transit key (which is passed in the authRequest token) is used to encrypt the app-private key. The ephemeral transit key is not itself passed back to the application (since the application should have already stored it during the authRequest generation).

The identity-address private key is used to sign the authResponse JWT, but the private key itself is not passed in the authResponse (only the public key is).

Reply:
The description in overview_auth.md's section "##Key pairs" is correct

Reply:
done.